src/behavior-considered-undefined.md

```diff
@@ -23,44 +23,56 @@ code.
 </div>
 
 * Data races.
-* Dereferencing a null or dangling raw pointer.
-* Unaligned pointer reading and writing outside of [`read_unaligned`]
-  and [`write_unaligned`].
-* Reads of [undef] \(uninitialized) memory.
-* Breaking the [pointer aliasing rules] on accesses through raw pointers;
```

Comment:
and removing this?

Reply:
It is still there, I just removed "on accesses through raw pointers" as that makes it sound like unsafe code doesn't have to worry when using references.